File: Beginner/AgeChecker.cpp
//Input The Age, if age > 18 print adult , if 10 < age <= 18 print teen, if age < 10 print child
int main(){
  int age;
  cin>>age;
  if(age>18){
    cout<<"Adult"<<endl;
  }
  else if(age<10){
    cout<<"Child"<<endl;
  }
  else{
    cout<<"Teen"<<endl;
  }
}

File: Beginner/AgeChecker.py
# Importing the Sys Module

import sys



def ageChecker(age):
    if(age >= 0):
        if(age > 18):
            return 'adult'
        elif(age > 10 and age <= 18):
            return 'teen'
        elif(age < 10 and age >= 0):
            return 'child'
    elif (age < 0):
        return "Age Shouldn't be Negative"


personAge = sys.stdin.readline()

personAge = int(personAge)

result = ageChecker(personAge)

print(result)